README.md

```diff
@@ -5,10 +5,13 @@ This repository collects a set of [Helm](https://helm.sh) charts curated by [Mat
 
 Click on the following links to see installation instructions for each chart:
 
-- [mattermost-team-edition](charts/mattermost-team-edition/)
+- [focalboard](charts/focalboard/)
+- [mattermost-chaos-engine](charts/mattermost-chaos-engine/)
 - [mattermost-enterprise-edition](charts/mattermost-enterprise-edition/)
-- [mattermost-push-proxy](charts/mattermost-push-proxy/)
 - [mattermost-operator](charts/mattermost-operator/)
+- [mattermost-push-proxy](charts/mattermost-push-proxy/)
+- [mattermost-rtcd](charts/mattermost-rtcd/)
+- [mattermost-team-edition](charts/mattermost-team-edition/)
 
 ## Usage
```

Comment:
Maybe add a section to document  locally testing/linting?

Reply:
Testing is mentioned in contributors guide since we have a specific document for that. I will add lint section.

![image](https://user-images.githubusercontent.com/98656635/182564994-d906ba1f-8619-419c-8663-6d3ffca65307.png)

Reply:
After second thought, better to have those in the REadme.md to improve local development experience. 

I moved those to the Readme.md as requested.